script/test

```diff
@@ -121,7 +126,7 @@ do_build()
 
 do_clean()
 {
-    rm -rfv "${OT_BUILDDIR}"
+    rm -rfv "${OT_BUILDDIR}" || sudo rm -rfv "${OT_BUILDDIR}"
```

Comment:
What about directly using `sudo`?

Reply:
Mostly sudo is not needed. Try it without sudo first to avoid input password.